### PR TITLE
Added db credentials by Spring

### PR DESCRIPTION
**Pull request by javsanbel2**

### CODE REVIEW

File: files/beekeeper-graphite-config-segment.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+,
+  "graphite": {
+    "enabled": "${graphite_enabled}"
+    "host": "${graphite_host}"
+    "prefix": "${graphite_prefix}"
+    "port": "${graphite_port}"
+  }
```
Comment (vedantchokshi) [inline files/beekeeper-graphite-config-segment.json:7]:
```suggestion
  }
  
```

File: k8s-config.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+resource "kubernetes_secret" "beekeeper_db_password_secret" {
+  metadata {
+    name      = var.k8s_db_password_secret
+    namespace = var.k8s_namespace
+  }
+
+  data = {
+    "${local.db_password_key}" = chomp(data.aws_secretsmanager_secret_version.beekeeper_db.secret_string)
+  }
+}
```
Comment (abhimanyugupta07) [inline k8s-config.tf:10]:
```suggestion
}

```

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Terraform deployment scripts for [Beekeeper](https://github.com/ExpediaGroup/bee
 Includes support for deploying Beekeeper on ECS and Kubernetes. Also includes deployment scripts for a Lambda which notifies Slack when Beekeeper's dead letter queue receives a message (this module is not required to run Beekeeper).
 
 ## Dependencies
-If the chosen `db_password_strategy` is `aws-secrets-manager`, this terraform module will not create the database password automatically. The secret will need to be created in Secrets Manager separately with the specified `db_password_key`.
+A database password defined in in `aws-secrets-manager`. The terraform module will use that password for the database (see property `db_password_key`).
```
Comment (abhimanyugupta07) [inline README.md:7]:
```suggestion
A database password is defined in `aws-secrets-manager`. The terraform module will use that password for the database (see property `db_password_key`).
```

File: README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ Terraform deployment scripts for [Beekeeper](https://github.com/ExpediaGroup/bee
 Includes support for deploying Beekeeper on ECS and Kubernetes. Also includes deployment scripts for a Lambda which notifies Slack when Beekeeper's dead letter queue receives a message (this module is not required to run Beekeeper).
 
 ## Dependencies
-If the chosen `db_password_strategy` is `aws-secrets-manager`, this terraform module will not create the database password automatically. The secret will need to be created in Secrets Manager separately with the specified `db_password_key`.
+A database password is defined in `aws-secrets-manager`. The terraform module will use that password for the database (see property `db_password_key`).
```
Comment (abhimanyugupta07) [inline README.md:7]:
```suggestion
A database password is defined in `aws-secrets-manager`. The terraform module will use that password for the database (see property `k8s_db_password_secret `).

```
?

File: k8s-path-cleanup.tf [hunk 5]
```diff
@@ -76,28 +76,28 @@ resource "kubernetes_deployment" "beekeeper_path_cleanup" {
           }
 
           env {
-            name  = "AWS_REGION"
+            name  = local.aws_region_key
```
Comment (abhimanyugupta07) [inline k8s-path-cleanup.tf:79]: why is this change required ?

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.1.2] - 2021-02-17
```
Comment (abhimanyugupta07) [inline CHANGELOG.md:6]: The date will need an update depending on when you are ready to release

File: k8s-path-cleanup.tf (hunk 5 above)
Comment (javsanbel2) [reply k8s-path-cleanup.tf:79]: We put it to refactor the code a little. We saw that the rest were using locals and that one was the only one that didn't use it.

Comment (abhimanyugupta07) [reply k8s-path-cleanup.tf:79]: ok.

File: CHANGELOG.md (hunk 6 above)
Comment (abhimanyugupta07) [reply CHANGELOG.md:6]: you want to put today's date here then ?

Comment (javsanbel2) [reply CHANGELOG.md:6]: yep, I was waiting until your answer so I commit the two changes at the same time

**Review by abhimanyugupta07 — APPROVED** [review]
